courses/csharp/05-Inheritance.md

```diff
@@ -17,4 +17,4 @@
     - post on 'CS Internship - C# Club' telegram groupe using #cs_internship #csharp tags.
  4. Exactly **7 days** before your deadline, ask your mentor to arrange a twitch time for you on your deadline day.
  5. You should give a 20-min presentation about the content of this step, on Twitch platform.
-
+ 6. You should become mentor to help another interns and learn more soft skills.

```

Comment:
6. It's time to be a mentor! Ask your mentor to start your onboarding process for mentorship.

Reply:
@nfsdns